### PR TITLE
Config with pure python

### PR DESCRIPTION
**Pull request by icemac**
Lint the code + fix line endings + Add support for Python 3.9.

### CODE REVIEW

File: src/z3c/authviewlet/session.py [hunk 1]
```diff
@@ -27,10 +29,7 @@ def __init__(self, *args):
             self.request.principal)
 
         camefrom = self.request.get('camefrom')
-        if isinstance(camefrom, list):
-            # this can happen on python2.6, as it changed the
-            # behaviour of cgi.FieldStorage a bit.
-            camefrom = camefrom[0]
+        assert isinstance(camefrom, basestring), type(camefrom)
```
Comment (mgedmin) [inline src/z3c/authviewlet/session.py]: Can't this happen on 2.7+ too, if the user edits the location bar and adds a second `&camefrom=somewhere-else-too`?

I don't think users should be able to trigger internal server errors by monkeying around with URLs.

File: src/z3c/authviewlet/tests/__init__.py [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
+# This software is subject to the provisions of the Zope Public License,
+# Version 2.1 (ZPL).  A copy of the ZPL should accompany this distribution.
+# THIS SOFTWARE IS PROVIDED "AS IS" AND ANY AND ALL EXPRESS OR IMPLIED
+# WARRANTIES ARE DISCLAIMED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF TITLE, MERCHANTABILITY, AGAINST INFRINGEMENT, AND FITNESS
+# FOR A PARTICULAR PURPOSE.
+#
+##############################################################################
+"""
+$Id:$
```
Comment (mgedmin) [inline src/z3c/authviewlet/tests/__init__.py]: What is going on here.  Did the file have DOS line endings?

(Also as long as we're changing everything, we might as well mass-delete the Subversion `$Id:$` placeholders.)

**Review by mgedmin — CHANGES_REQUESTED** [review]
That rewrite of `if isinstance():` to an `assert` looks suspicious to me and I'd rather not have it in this PR.

File: src/z3c/authviewlet/tests/__init__.py (hunk 2 above)
Comment (mgedmin) [reply src/z3c/authviewlet/tests/__init__.py]: (Yeah, the diff becomes nice and readable after asking GH to hide whitespace changes.  The point about `$Id:$` stands.)

File: src/z3c/authviewlet/session.py (hunk 1 above)
Comment (icemac) [reply src/z3c/authviewlet/session.py]: You are right, I was misguided by the comment. I am going to switch back to the original code and add a test.

File: src/z3c/authviewlet/tests/__init__.py (hunk 2 above)
Comment (icemac) [reply src/z3c/authviewlet/tests/__init__.py]: Yes the files actually had DOS line endings. I am going to remove the SVN leftovers.

Comment (icemac) [reply src/z3c/authviewlet/tests/__init__.py]: Hm, it was just this single one.

**Review by mgedmin — APPROVED** [review]
LGTM!

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**